### PR TITLE
[Merged by Bors] - feat: Add `mul_inv` and `inv_mul` versions of `div_le_one_of_le`

### PR DESCRIPTION
**Pull request by girving**
These are exactly `div_le_one_of_le` with `a / b` replaced by `a * b⁻¹` and `b⁻¹ * a`.

They come up frequently because some tactics produce inverses as normal forms over division.  The iff versions with `0 < b` already exist as `mul_inv_le_one_iff` and `inv_mul_le_one_iff`; the point of these is to have a weaker nonnegativity assumption on `b`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by girving** [comment]
Possibly I should add `inv_mul_le_one_of_le` too?

**Comment by YaelDillies** [comment]
Yes, and iff versions assuming `0 < b`. Also please edit the PR title, I cannot comprehend it 😄

**Comment by girving** [comment]
Is that title better?

I'll add the inv_mul version now.  The iff versions already exist: [mul inv](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Order/Group/Defs.html#mul_inv_le_one_iff), [inv mul](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Order/Group/Defs.html#inv_mul_le_one_iff).

**Comment by YaelDillies** [comment]
Now that's better!

**Comment by girving** [comment]
@YaelDillies: Would you be up for a review and approval if it seems reasonable?

File: Mathlib/Algebra/Order/Field/Basic.lean
Comment (YaelDillies) [inline Mathlib/Algebra/Order/Field/Basic.lean:250]: Those docstrings read like "Here's the reason I made a PR", which is not very useful on declarations in Lean code. They would fit better in your PR description
```suggestion
lemma mul_inv_le_one_of_le (h : a ≤ b) (hb : 0 ≤ b) : a * b⁻¹ ≤ 1 := by
  simpa only [← div_eq_mul_inv] using div_le_one_of_le h hb

```

**Review by YaelDillies — APPROVED** [review]
Thanks!

maintainer merge

**Comment by girving** [comment]
What’s the next step after “maintainer merge”?

**Comment by YaelDillies** [comment]
Nothing on your part. A maintainer will merge the PR or come up with a comment I hadn't thought of.

**Comment by urkud** [comment]
Thanks! :tada:
bors merge

**Pull request closed without merging**
